qiskit/test/mock/utils/backend_converter.py

```diff
@@ -0,0 +1,140 @@
+            gates[name][qubits] = InstructionProperties(**gate_props)
+        for gate, props in gates.items():
+            if gate in name_mapping:
+                inst = name_mapping[gate]
+            else:
+                inst = custom_gates[gate]
+            target.add_instruction(inst, props)
+        # Create measurement instructions:
+        measure_props = {}
+        count = 0
+        for qubit in props_dict["qubits"]:
+            qubit_prop = {}
+            for prop in qubit:
+                if prop["name"] == "readout_length":
+                    qubit_prop["duration"] = apply_prefix(prop["value"], prop["unit"])
+                if prop["name"] == "readout_error":
+                    qubit_prop["error"] = prop["value"]
+            measure_props[(count,)] = InstructionProperties(**qubit_prop)
+            count += 1
+        target.add_instruction(Measure(), measure_props)
+    # Parse from configuration because properties doesn't exist
+    else:
+        for gate in conf_dict["gates"]:
+            name = gate["name"]
+            gate_props = {tuple(x): None for x in gate["coupling_map"]}
+            if name in name_mapping:
+                target.add_instruction(name_mapping[name], gate_props)
+            else:
+                custom_gate = Gate(name, len(gate["coupling_map"][0]))
+                target.add_instruction(custom_gate, gate_props)
+        measure_props = {(n,): None for n in range(conf_dict["n_qubits"])}
+        target.add_instruction(Measure(), measure_props)
+    # parse global configuration properties
+    dt = conf_dict.get("dt")
+    if dt:
+        target.dt = dt
+    if "timing_constraints" in conf_dict:
+        target.granularity = conf_dict["timing_constraints"].get("granularity")
+        target.min_length = conf_dict["timing_constraints"].get("min_length")
+        target.pulse_alignment = conf_dict["timing_constraints"].get("pulse_alignment")
+        target.aquire_alignment = conf_dict["timing_constraints"].get("acquire_alignment")
+    # If pulse defaults exists use that as the source of truth
+    if defs_dict is not None:
+        pulse_defs = PulseDefaults.from_dict(defs_dict)
+        inst_map = pulse_defs.instruction_schedule_map
+        for inst in inst_map.instructions:
+            for qarg in inst_map.qubits_with_instruction(inst):
+                sched = inst_map.get(inst, qarg)
+                if inst in target:
+                    try:
+                        qarg = tuple(qarg)
+                    except TypeError:
+                        qarg = (qarg,)
+                    if inst == "measure":
+                        for qubit in qarg:
+                            target[inst][(qubit,)].calibration = sched
+                    else:
+                        target[inst][qarg].calibration = sched
+    return target
+
+
+def qubit_props_dict_from_props_dict(properties: dict) -> dict:
```

Comment:
pylint gives warning that this function is invalid, probably because it has more than 30 characters. Would you suggest to rename it to something shorter? @mtreinish https://dev.azure.com/qiskit-ci/qiskit-terra/_build/results?buildId=34886&view=logs&j=f0ab2c38-58fa-55ad-8e8b-d3984171f737&t=980625fc-4a64-50e4-7ca0-679c2a4bbda8&l=34

Reply:
How about `qubit_dict_from_props`?

Reply:
`qubit_dict` in `qubit_dict_from_props` might be confusing. How about `qubit_props_from_props`?

Reply:
I am using `qubit_props_from_props` in the latest commit. Please let me know if you have objections.